diesel_cli/src/cli.rs

```diff
@@ -162,6 +162,10 @@ pub fn build_cli() -> App<'static, 'static> {
                 .possible_values(&Shell::variants()),
         );
 
+    let features_subcommand = SubCommand::with_name("features").about(
+        "list support for which database engines have been compiled into this version of diesel"
```

Comment:
```suggestion
        "lists all the database engines available with the version of diesel the user is using"
```

Maybe better?

Reply:
I agree. Thank you for the suggestion. Your sentence is much easier to parse and sounds more vernacular. I've tweaked it a little to make it shorter.